src/main/java/org/numenta/nupic/network/Region.java

```diff
@@ -457,6 +457,9 @@ Region connect(Region inputRegion) {
             @Override public void onError(Throwable e) { e.printStackTrace(); }
             @SuppressWarnings("unchecked")
             @Override public void onNext(Inference i) {
+                // TODO: 21: This is where classifierInput is set. Need to change
+                // it to respect only fields user has specified for classification
+                // with the new Parameters KEY.
```

Comment:
@Hopding Has this been done? Or is this old documentation (reminders)? If so can you remove these comments? If not, would this not need to be done?